packages/internal/bridge/sdk/src/tokenBridge.ts

```diff
@@ -110,12 +148,16 @@ export class TokenBridge {
    *   });
    */
   public async getFee(req: BridgeFeeRequest): Promise<BridgeFeeResponse> {
-    await this.validateChainConfiguration();
-
-    if (req.action !== BridgeFeeActions.FINALISE_WITHDRAWAL) {
-      await this.validateChainIds(req.sourceChainId, req.destinationChainId);
-    }
-    return await this.getFeePrivate(req);
+    const [, , res] = await Promise.all([
+      this.initialise(),
```

Comment:
Yes, the intention is to avoid any breaking change and how this SDK is being used. Calling `initialise` in each of the calls makes sure this instance is initialised.

Reply:
Yep that makes sense.